src/components/Login/LoginContainer.tsx

```diff
@@ -130,18 +133,11 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
       if (!lastResendWarningMessageRef.current) {
         resolve(true);
       } else {
-        showAlert({
-          ...defaultConfirmationProps,
-          title: "Resend OTP?",
-          description: lastResendWarningMessageRef.current,
-          buttonTexts: {
-            primaryActionText: "Resend",
-            secondaryActionText: "No"
-          },
-          onOk: () => resolve(true),
-          onCancel: () => resolve(false),
-          visible: true
-        });
+        showConfirmationAlert(
+          new Error("resendOTP"),
```

Comment:
Same

Reply:
This has been refactored to:
```suggestion
         showConfirmationAlert(
          CONFIRMATION_MESSAGE.RESEND_OTP,
```